### PR TITLE
[Remote-PT-0620] debora Crescenzo

### PR DESCRIPTION
**Pull request by NubivagantYansa**
Done

### CODE REVIEW

File: queries.md [hunk 1]
```diff
@@ -4,76 +4,96 @@
...
+filter: {number_of_employees: {\$gt: 5000} }
+project:
+sort: {number_of_employees:1}
+Collaboration:
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{founded_year:{$gte:2000}}, {founded_year:{\$lte:2005}}]}
```
Comment (itstheandre) [inline queries.md:20]: You can save yourself here some keystrokes:
```js
founded_year: {  $gte: 2000, $lte: 2005  }
```

File: queries.md [hunk 2]
```diff
@@ -4,76 +4,96 @@
...
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{number_of_employees:{$lt:1000}}, {founded_year:{\$lt:2005}}]}
+sort: {number_of_employees: 1}
+limit: 10
 
 ### 6. All the companies that don't include the `partners` field.
 
-<!-- Your Code Goes Here -->
+find: {partners: {\$size:0}}
```
Comment (itstheandre) [inline queries.md:38]: more correct here would be:
```js
 {partners: {$exists: false}}
```

File: queries.md [hunk 3]
```diff
@@ -4,76 +4,96 @@
...
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
 
-<!-- Your Code Goes Here -->
+Find: {category_code:null}
```
Comment (itstheandre) [inline queries.md:42]: more correct "Mongowise" would be:
```js
 {category_code: {$type: 'null'}}
```

File: queries.md (hunk 1 above)
Comment (NubivagantYansa) [reply queries.md:20]: Brill!

File: queries.md (hunk 2 above)
Comment (NubivagantYansa) [reply queries.md:38]: I see!! I found that, but thought the array bit was more JS-like

**Pull request closed without merging**